doc/cephfs/administration.rst

```diff
@@ -92,6 +92,14 @@ The CephX IDs authorized to the old file system name need to be reauthorized
 to the new name. Any on-going operations of the clients using these IDs may be
 disrupted. Mirroring is expected to be disabled on the file system.
 
+::
+
+    fs swap <fs1-name> <fs2-name> [--swap-fscids] [--yes-i-really-mean-it]
+
+Swaps names of two Ceph file sytems and updates the application tags on all
+pools of both FSs accordingly. Passing ``swap-fscids`` swaps FSCIDs as well.
+Mirroring should be disabled on both the FSs. CephX IDs needs to reauthorized
```

Comment:
> Mirroring should be disabled on both the FSs.

Why? (explain!)

Reply:
if you could add a suggestion (about explanation you want to see) along, we just might be done in one less cycle of PR review. :)

Reply:
I believe it's because mirroring uses the fscid. Ramana discovered this but may not have left an explanation in the docs for `fs rename`.

Reply:
I had suggested @ajarr that file system mirroring should be disabled before renaming a file system. The same explanation applies here as well.

Reply:
So, the check needs to be done before we initiate file system mirroring and needs to be updated in the docs.

Reply:
> So, the check needs to be done before we initiate file system mirroring and needs to be updated in the docs.

I get that but can you elaborate on the part why this is needed? Patrick wants us to add that to the doc.

Reply:
The cephfs-mirror daemon uses fscid internally and changing the fscid without the mirror daemon knowing could result in undefined behaviour.

Reply:
> The cephfs-mirror daemon uses fscid internally and changing the fscid without the mirror daemon knowing could result in undefined behaviour.

Got it. Thank you so much!

Reply:
Uh, please add the explanation to the docs!!

Reply:
Yep.